sdk/devcenter/Azure.Developer.DevCenter/src/Generated/DevBoxesClient.cs

```diff
@@ -0,0 +1,2704 @@
+        ///   provisioningState: string, # Optional. The current provisioning state of the Dev Box.
+        ///   actionState: string, # Optional. The current action state of the Dev Box. This is state is based on previous action performed by user.
+        ///   powerState: &quot;Unknown&quot; | &quot;Deallocated&quot; | &quot;PoweredOff&quot; | &quot;Running&quot; | &quot;Hibernated&quot;, # Optional. The current power state of the Dev Box.
+        ///   uniqueId: string, # Optional. A unique identifier for the Dev Box. This is a GUID-formatted string (e.g. 00000000-0000-0000-0000-000000000000).
+        ///   errorDetails: {
+        ///     code: string, # Optional. The error code.
+        ///     message: string, # Optional. The error message.
+        ///   }, # Optional. Provisioning or action error details. Populated only for error states.
+        ///   location: string, # Optional. Azure region where this Dev Box is located. This will be the same region as the Virtual Network it is attached to.
+        ///   osType: &quot;Windows&quot;, # Optional. The operating system type of this Dev Box.
+        ///   user: string, # Optional. User identifier of the user this vm is assigned to.
+        ///   hardwareProfile: {
+        ///     skuName: string, # Optional. The name of the SKU
+        ///     vCPUs: number, # Optional. The number of vCPUs available for the Dev Box.
+        ///     memoryGB: number, # Optional. The amount of memory available for the Dev Box.
+        ///   }, # Optional. Information about the Dev Box&apos;s hardware resources
+        ///   storageProfile: {
+        ///     osDisk: {
+        ///       diskSizeGB: number, # Optional. The size of the OS Disk in gigabytes.
+        ///     }, # Optional. Settings for the operating system disk.
+        ///   }, # Optional. Storage settings for this Dev Box
+        ///   imageReference: {
+        ///     name: string, # Optional. The name of the image used.
+        ///     version: string, # Optional. The version of the image.
+        ///     operatingSystem: string, # Optional. The operating system of the image.
+        ///     osBuildNumber: string, # Optional. The operating system build number of the image.
+        ///     publishedDate: string (ISO 8601 Format), # Optional. The datetime that the backing image version was published.
+        ///   }, # Optional. Information about the image used for this Dev Box
+        ///   createdTime: string (ISO 8601 Format), # Optional. Creation time of this Dev Box
+        ///   localAdministrator: &quot;Enabled&quot; | &quot;Disabled&quot;, # Optional. Indicates whether the owner of the Dev Box is a local administrator.
+        /// }
+        /// </code>
+        /// 
+        /// </remarks>
+        public virtual async Task<Response> GetDevBoxByUserAsync(string devBoxName, string userId = "me", RequestContext context = null)
```

Comment:
@fengzhou-msft, @chunyu3 and @pshao25 - Most of the generated protocol methods in this class is failing with `Error CS1737: Optional parameters must appear after all required parameters`.

Could you please look into that?

Reply:
See all the failures here: https://github.com/Azure/azure-sdk-for-net/pull/31636/checks?check_run_id=8854091873